spec/board_spec.rb

```diff
@@ -0,0 +1,68 @@
+# spec :board_spec.rb
+require './lib/board.rb'
+
+RSpec.describe Board do
+  let(:board) { Board.new }
+  describe '.valid?' do
+    context 'when user gives input' do
+      it 'returns true if this input is in range of 1..9 and this input is not taken' do
+        expect(board.valid?([1, 2])).to eq true
+      end
+
+      it 'returns false if this input is taken' do
+        board.update([1, 2])
```

Comment:
- [ ] **[OPTIONAL]** What if `update` method is buggy? the test will fail, not because object under test (`valid?`) is not working right, it's because your test depends on another method which is not working. I suggest that you use `board.push([1, 2])` instead of depending on another method.